user/advanced-topics/guivm.md

```diff
@@ -0,0 +1,134 @@
+---
+lang: en
+layout: doc
+permalink: /doc/guivm-configuration/
+ref: 184
+title: GuiVM Configuration
+---
+
+## Gui domain
+
+In this section, we describe how to setup `GuiVM` in several case as described in [GUI Domain](https://www.qubes-os.org/news/2020/03/18/gui-domain/). In all the cases, the base underlying TemplateVM used is `Fedora` with `XFCE` flavor to match current desktop choice in `dom0`. That can be adapted very easily for other desktops and templates. By default, the configured `GuiVM` is a management qube with global admin permissions `rwx` but can be adjusted to `ro` (see [Introducing the Qubes Admin API](https://www.qubes-os.org/news/2017/06/27/qubes-admin-api/)) in pillar data of the corresponding `GuiVM` to setup. Please note that each `GuiVM` has no `NetVM`.
```

Comment:
When mentioning the change to `ro` admin rights, this should at least mention in which file to make the change. Digging through Salt documentation is not easy...

Reply:
Sure, that's on the improvement to be done.